_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,473 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.field-group {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .field-group-hd {
+        margin-bottom: spacing-vertical(small);
+        font-size: font-size(mid-large);
+    }
+
+    .field {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $field-tip-color;
+        }
+    }
+
+    &.form-group {
+
+        .field {
+            margin-bottom: spacing-vertical(x-small);
+        }
+
+        .helper-text {
+            margin-top: spacing-vertical(xx-small);
+            @include margin-left(spacing-horizontal(mid-large));
+        }
+    }
+}
+
+.form-label {
+    display: block;
+    width: auto;
+    margin-bottom: spacing-vertical(x-small);
+    font-size: font-size(base);
+    line-height: 100%;
+    color: $label-color;
+
+    // presents the label inline with the form control
+    &.label-inline {
+        display: inline-block;
+        margin-bottom: 0;
+    }
+
+    // STATE: is selected
+    .form-input:checked + .form-input-label,
+    .form-radio:checked + .form-input-label,
+    &.is-active,
+    &.is-selected {
+        @include transition(all timing(xx-fast) ease-in-out 0s);
+        color: $label-color-active;
+    }
+}
+
+.helper-text {
+    margin-top: spacing-vertical(x-small);
+}
+
+.field-required:after {
```

Comment:
Just confirming that this will work with RTL directional changes (the asterisk will be displayed to the left of the label), right?